src/utils/paths.ts

```diff
@@ -1,17 +1,19 @@
-interface Paths {
+type Paths = {
```

Comment:
In TS for performance reasons (boost compilation) better to use interfaces instead of types...